Failed to deploy on github

### Have you checked that your issue isn't already filed?

- [x] I read through [FAQ](https://github.com/alshedivat/al-folio/blob/main/FAQ.md) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my issue.
- [x] Yes, I have checked that this issue isn't already filed.

### Bug description

Github Action failed at deploy stage. The repo is https://github.com/fandaren/fandaren.github.io

### How to reproduce the bug

I followed the Recommended Approach in [instructions](https://github.com/alshedivat/al-folio/blob/main/INSTALL.md). The only modification to the code was to change url to github username and left baseurl empty. The problem happened at step 4. 


### Error messages and logs

The error message at install and build session of deploy site action was:
```
Run sudo apt-get update && sudo apt-get install -y imagemagick
  sudo apt-get update && sudo apt-get install -y imagemagick
  pip3 install --upgrade nbconvert
  export JEKYLL_ENV=production
  bundle exec jekyll build
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.13.9/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.13.9/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.9/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.9/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.9/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.13.9/x64/lib
Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [144 B]
Get:6 https://packages.microsoft.com/repos/azure-cli noble InRelease [3564 B]
Hit:2 http://azure.archive.ubuntu.com/ubuntu noble InRelease
Get:7 https://packages.microsoft.com/ubuntu/24.04/prod noble InRelease [3600 B]
Get:3 http://azure.archive.ubuntu.com/ubuntu noble-updates InRelease [126 kB]
Get:4 http://azure.archive.ubuntu.com/ubuntu noble-backports InRelease [126 kB]
Get:5 http://azure.archive.ubuntu.com/ubuntu noble-security InRelease [126 kB]
Get:8 https://packages.microsoft.com/repos/azure-cli noble/main amd64 Packages [1763 B]
Get:9 https://packages.microsoft.com/ubuntu/24.04/prod noble/main amd64 Packages [68.1 kB]
Get:10 https://packages.microsoft.com/ubuntu/24.04/prod noble/main armhf Packages [11.2 kB]
Get:11 https://packages.microsoft.com/ubuntu/24.04/prod noble/main arm64 Packages [50.9 kB]
Get:12 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 Packages [1588 kB]
Get:13 http://azure.archive.ubuntu.com/ubuntu noble-updates/main Translation-en [300 kB]
Get:14 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 Components [175 kB]
Get:15 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 c-n-f Metadata [15.7 kB]
Get:16 http://azure.archive.ubuntu.com/ubuntu noble-updates/universe amd64 Packages [1500 kB]
Get:17 http://azure.archive.ubuntu.com/ubuntu noble-updates/universe Translation-en [303 kB]
Get:18 http://azure.archive.ubuntu.com/ubuntu noble-updates/universe amd64 Components [378 kB]
Get:19 http://azure.archive.ubuntu.com/ubuntu noble-updates/universe amd64 c-n-f Metadata [31.4 kB]
Get:20 http://azure.archive.ubuntu.com/ubuntu noble-updates/restricted amd64 Packages [2251 kB]
Get:21 http://azure.archive.ubuntu.com/ubuntu noble-updates/restricted Translation-en [510 kB]
Get:22 http://azure.archive.ubuntu.com/ubuntu noble-updates/restricted amd64 Components [212 B]
Get:23 http://azure.archive.ubuntu.com/ubuntu noble-updates/multiverse Translation-en [5808 B]
Get:24 http://azure.archive.ubuntu.com/ubuntu noble-updates/multiverse amd64 Components [940 B]
Get:25 http://azure.archive.ubuntu.com/ubuntu noble-backports/main amd64 Components [7156 B]
Get:26 http://azure.archive.ubuntu.com/ubuntu noble-backports/universe amd64 Components [10.9 kB]
Get:27 http://azure.archive.ubuntu.com/ubuntu noble-backports/restricted amd64 Components [216 B]
Get:28 http://azure.archive.ubuntu.com/ubuntu noble-backports/multiverse amd64 Components [212 B]
Get:29 http://azure.archive.ubuntu.com/ubuntu noble-security/main amd64 Packages [1311 kB]
Get:30 http://azure.archive.ubuntu.com/ubuntu noble-security/main Translation-en [216 kB]
Get:31 http://azure.archive.ubuntu.com/ubuntu noble-security/main amd64 Components [21.6 kB]
Get:32 http://azure.archive.ubuntu.com/ubuntu noble-security/main amd64 c-n-f Metadata [9420 B]
Get:33 http://azure.archive.ubuntu.com/ubuntu noble-security/universe amd64 Packages [909 kB]
Get:34 http://azure.archive.ubuntu.com/ubuntu noble-security/universe Translation-en [205 kB]
Get:35 http://azure.archive.ubuntu.com/ubuntu noble-security/universe amd64 Components [52.2 kB]
Get:36 http://azure.archive.ubuntu.com/ubuntu noble-security/universe amd64 c-n-f Metadata [19.4 kB]
Get:37 http://azure.archive.ubuntu.com/ubuntu noble-security/restricted amd64 Packages [2146 kB]
Get:38 http://azure.archive.ubuntu.com/ubuntu noble-security/restricted Translation-en [488 kB]
Get:39 http://azure.archive.ubuntu.com/ubuntu noble-security/restricted amd64 Components [212 B]
Get:40 http://azure.archive.ubuntu.com/ubuntu noble-security/multiverse Translation-en [5956 B]
Get:41 http://azure.archive.ubuntu.com/ubuntu noble-security/multiverse amd64 Components [212 B]
Fetched 13.0 MB in 2s (7640 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  fonts-droid-fallback fonts-noto-mono fonts-urw-base35 ghostscript
  imagemagick-6.q16 libdjvulibre-text libdjvulibre21 libgs-common libgs10
  libgs10-common libidn12 libijs-0.35 libimath-3-1-29t64 libjbig2dec0
  libjxr-tools libjxr0t64 libmagickcore-6.q16-7-extra libnetpbm11t64
  libopenexr-3-1-30 libpaper-utils libpaper1 libwmflite-0.2-7 netpbm
  poppler-data xfonts-encodings xfonts-utils
Suggested packages:
  fonts-noto fonts-freefont-otf | fonts-freefont-ttf fonts-texgyre
  texlive-binaries imagemagick-6-doc autotrace cups-bsd | lpr | lprng enscript
  ffmpeg gimp gnuplot grads graphviz hp2xx html2ps libwmf-bin mplayer povray
  radiance sane-utils texlive-base-bin transfig libraw-bin inkscape
  poppler-utils fonts-japanese-mincho | fonts-ipafont-mincho
  fonts-japanese-gothic | fonts-ipafont-gothic fonts-arphic-ukai
  fonts-arphic-uming fonts-nanum
The following NEW packages will be installed:
  fonts-droid-fallback fonts-noto-mono fonts-urw-base35 ghostscript
  imagemagick imagemagick-6.q16 libdjvulibre-text libdjvulibre21 libgs-common
  libgs10 libgs10-common libidn12 libijs-0.35 libimath-3-1-29t64 libjbig2dec0
  libjxr-tools libjxr0t64 libmagickcore-6.q16-7-extra libnetpbm11t64
  libopenexr-3-1-30 libpaper-utils libpaper1 libwmflite-0.2-7 netpbm
  poppler-data xfonts-encodings xfonts-utils
0 upgraded, 27 newly installed, 0 to remove and 45 not upgraded.
Need to get 25.2 MB of archives.
After this operation, 85.4 MB of additional disk space will be used.
Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [144 B]
Get:2 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 fonts-droid-fallback all 1:6.0.1r16-1.1build1 [1805 kB]
Get:3 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 poppler-data all 0.4.12-1 [2060 kB]
Get:4 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 fonts-noto-mono all 20201225-2 [435 kB]
Get:5 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 xfonts-encodings all 1:1.0.5-0ubuntu2 [578 kB]
Get:6 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 xfonts-utils amd64 1:7.7+6build3 [94.4 kB]
Get:7 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 fonts-urw-base35 all 20200910-8 [11.0 MB]
Get:8 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 libgs-common all 10.02.1~dfsg1-0ubuntu7.8 [176 kB]
Get:9 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 libgs10-common all 10.02.1~dfsg1-0ubuntu7.8 [488 kB]
Get:10 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 libidn12 amd64 1.42-1build1 [55.9 kB]
Get:11 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 libijs-0.35 amd64 0.35-15.1build1 [15.3 kB]
Get:12 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 libjbig2dec0 amd64 0.20-1build3 [65.0 kB]
Get:13 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 libpaper1 amd64 1.1.29build1 [13.4 kB]
Get:14 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 libgs10 amd64 10.02.1~dfsg1-0ubuntu7.8 [3897 kB]
Get:15 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 ghostscript amd64 10.02.1~dfsg1-0ubuntu7.8 [43.4 kB]
Get:16 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 imagemagick-6.q16 amd64 8:6.9.12.98+dfsg1-5.2build2 [254 kB]
Get:17 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 imagemagick amd64 8:6.9.12.98+dfsg1-5.2build2 [14.2 kB]
Get:18 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 libdjvulibre-text all 3.5.28-2ubuntu0.24.04.1 [50.9 kB]
Get:19 http://azure.archive.ubuntu.com/ubuntu noble-updates/main amd64 libdjvulibre21 amd64 3.5.28-2ubuntu0.24.04.1 [638 kB]
Get:20 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 libimath-3-1-29t64 amd64 3.1.9-3.1ubuntu2 [72.2 kB]
Get:21 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 libjxr0t64 amd64 1.2~git20170615.f752187-5.1ubuntu2 [181 kB]
Get:22 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 libjxr-tools amd64 1.2~git20170615.f752187-5.1ubuntu2 [15.8 kB]
Get:23 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 libopenexr-3-1-30 amd64 3.1.5-5.1build3 [1004 kB]
Get:24 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 libwmflite-0.2-7 amd64 0.2.13-1.1build3 [68.6 kB]
Get:25 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 libmagickcore-6.q16-7-extra amd64 8:6.9.12.98+dfsg1-5.2build2 [70.1 kB]
Get:26 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 libnetpbm11t64 amd64 2:11.05.02-1.1build1 [114 kB]
Get:27 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 libpaper-utils amd64 1.1.29build1 [8650 B]
Get:28 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 netpbm amd64 2:11.05.02-1.1build1 [2054 kB]
Preconfiguring packages ...
Fetched 25.2 MB in 1s (26.9 MB/s)
Selecting previously unselected package fonts-droid-fallback.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 216225 files and directories currently installed.)
Preparing to unpack .../00-fonts-droid-fallback_1%3a6.0.1r16-1.1build1_all.deb ...
Unpacking fonts-droid-fallback (1:6.0.1r16-1.1build1) ...
Selecting previously unselected package poppler-data.
Preparing to unpack .../01-poppler-data_0.4.12-1_all.deb ...
Unpacking poppler-data (0.4.12-1) ...
Selecting previously unselected package fonts-noto-mono.
Preparing to unpack .../02-fonts-noto-mono_20201225-2_all.deb ...
Unpacking fonts-noto-mono (20201225-2) ...
Selecting previously unselected package xfonts-encodings.
Preparing to unpack .../03-xfonts-encodings_1%3a1.0.5-0ubuntu2_all.deb ...
Unpacking xfonts-encodings (1:1.0.5-0ubuntu2) ...
Selecting previously unselected package xfonts-utils.
Preparing to unpack .../04-xfonts-utils_1%3a7.7+6build3_amd64.deb ...
Unpacking xfonts-utils (1:7.7+6build3) ...
Selecting previously unselected package fonts-urw-base35.
Preparing to unpack .../05-fonts-urw-base35_20200910-8_all.deb ...
Unpacking fonts-urw-base35 (20200910-8) ...
Selecting previously unselected package libgs-common.
Preparing to unpack .../06-libgs-common_10.02.1~dfsg1-0ubuntu7.8_all.deb ...
Unpacking libgs-common (10.02.1~dfsg1-0ubuntu7.8) ...
Selecting previously unselected package libgs10-common.
Preparing to unpack .../07-libgs10-common_10.02.1~dfsg1-0ubuntu7.8_all.deb ...
Unpacking libgs10-common (10.02.1~dfsg1-0ubuntu7.8) ...
Selecting previously unselected package libidn12:amd64.
Preparing to unpack .../08-libidn12_1.42-1build1_amd64.deb ...
Unpacking libidn12:amd64 (1.42-1build1) ...
Selecting previously unselected package libijs-0.35:amd64.
Preparing to unpack .../09-libijs-0.35_0.35-15.1build1_amd64.deb ...
Unpacking libijs-0.35:amd64 (0.35-15.1build1) ...
Selecting previously unselected package libjbig2dec0:amd64.
Preparing to unpack .../10-libjbig2dec0_0.20-1build3_amd64.deb ...
Unpacking libjbig2dec0:amd64 (0.20-1build3) ...
Selecting previously unselected package libpaper1:amd64.
Preparing to unpack .../11-libpaper1_1.1.29build1_amd64.deb ...
Unpacking libpaper1:amd64 (1.1.29build1) ...
Selecting previously unselected package libgs10:amd64.
Preparing to unpack .../12-libgs10_10.02.1~dfsg1-0ubuntu7.8_amd64.deb ...
Unpacking libgs10:amd64 (10.02.1~dfsg1-0ubuntu7.8) ...
Selecting previously unselected package ghostscript.
Preparing to unpack .../13-ghostscript_10.02.1~dfsg1-0ubuntu7.8_amd64.deb ...
Unpacking ghostscript (10.02.1~dfsg1-0ubuntu7.8) ...
Selecting previously unselected package imagemagick-6.q16.
Preparing to unpack .../14-imagemagick-6.q16_8%3a6.9.12.98+dfsg1-5.2build2_amd64.deb ...
Unpacking imagemagick-6.q16 (8:6.9.12.98+dfsg1-5.2build2) ...
Selecting previously unselected package imagemagick.
Preparing to unpack .../15-imagemagick_8%3a6.9.12.98+dfsg1-5.2build2_amd64.deb ...
Unpacking imagemagick (8:6.9.12.98+dfsg1-5.2build2) ...
Selecting previously unselected package libdjvulibre-text.
Preparing to unpack .../16-libdjvulibre-text_3.5.28-2ubuntu0.24.04.1_all.deb ...
Unpacking libdjvulibre-text (3.5.28-2ubuntu0.24.04.1) ...
Selecting previously unselected package libdjvulibre21:amd64.
Preparing to unpack .../17-libdjvulibre21_3.5.28-2ubuntu0.24.04.1_amd64.deb ...
Unpacking libdjvulibre21:amd64 (3.5.28-2ubuntu0.24.04.1) ...
Selecting previously unselected package libimath-3-1-29t64:amd64.
Preparing to unpack .../18-libimath-3-1-29t64_3.1.9-3.1ubuntu2_amd64.deb ...
Unpacking libimath-3-1-29t64:amd64 (3.1.9-3.1ubuntu2) ...
Selecting previously unselected package libjxr0t64:amd64.
Preparing to unpack .../19-libjxr0t64_1.2~git20170615.f752187-5.1ubuntu2_amd64.deb ...
Unpacking libjxr0t64:amd64 (1.2~git20170615.f752187-5.1ubuntu2) ...
Selecting previously unselected package libjxr-tools.
Preparing to unpack .../20-libjxr-tools_1.2~git20170615.f752187-5.1ubuntu2_amd64.deb ...
Unpacking libjxr-tools (1.2~git20170615.f752187-5.1ubuntu2) ...
Selecting previously unselected package libopenexr-3-1-30:amd64.
Preparing to unpack .../21-libopenexr-3-1-30_3.1.5-5.1build3_amd64.deb ...
Unpacking libopenexr-3-1-30:amd64 (3.1.5-5.1build3) ...
Selecting previously unselected package libwmflite-0.2-7:amd64.
Preparing to unpack .../22-libwmflite-0.2-7_0.2.13-1.1build3_amd64.deb ...
Unpacking libwmflite-0.2-7:amd64 (0.2.13-1.1build3) ...
Selecting previously unselected package libmagickcore-6.q16-7-extra:amd64.
Preparing to unpack .../23-libmagickcore-6.q16-7-extra_8%3a6.9.12.98+dfsg1-5.2build2_amd64.deb ...
Unpacking libmagickcore-6.q16-7-extra:amd64 (8:6.9.12.98+dfsg1-5.2build2) ...
Selecting previously unselected package libnetpbm11t64:amd64.
Preparing to unpack .../24-libnetpbm11t64_2%3a11.05.02-1.1build1_amd64.deb ...
Unpacking libnetpbm11t64:amd64 (2:11.05.02-1.1build1) ...
Selecting previously unselected package libpaper-utils.
Preparing to unpack .../25-libpaper-utils_1.1.29build1_amd64.deb ...
Unpacking libpaper-utils (1.1.29build1) ...
Selecting previously unselected package netpbm.
Preparing to unpack .../26-netpbm_2%3a11.05.02-1.1build1_amd64.deb ...
Unpacking netpbm (2:11.05.02-1.1build1) ...
Setting up libpaper1:amd64 (1.1.29build1) ...

Creating config file /etc/papersize with new version
Setting up fonts-noto-mono (20201225-2) ...
Setting up libwmflite-0.2-7:amd64 (0.2.13-1.1build3) ...
Setting up libijs-0.35:amd64 (0.35-15.1build1) ...
Setting up libgs-common (10.02.1~dfsg1-0ubuntu7.8) ...
Setting up libpaper-utils (1.1.29build1) ...
Setting up libnetpbm11t64:amd64 (2:11.05.02-1.1build1) ...
Setting up poppler-data (0.4.12-1) ...
Setting up libjxr0t64:amd64 (1.2~git20170615.f752187-5.1ubuntu2) ...
Setting up libjbig2dec0:amd64 (0.20-1build3) ...
Setting up xfonts-encodings (1:1.0.5-0ubuntu2) ...
Setting up libimath-3-1-29t64:amd64 (3.1.9-3.1ubuntu2) ...
Setting up libidn12:amd64 (1.42-1build1) ...
Setting up netpbm (2:11.05.02-1.1build1) ...
Setting up imagemagick-6.q16 (8:6.9.12.98+dfsg1-5.2build2) ...
update-alternatives: using /usr/bin/compare-im6.q16 to provide /usr/bin/compare (compare) in auto mode
update-alternatives: using /usr/bin/compare-im6.q16 to provide /usr/bin/compare-im6 (compare-im6) in auto mode
update-alternatives: using /usr/bin/animate-im6.q16 to provide /usr/bin/animate (animate) in auto mode
update-alternatives: using /usr/bin/animate-im6.q16 to provide /usr/bin/animate-im6 (animate-im6) in auto mode
update-alternatives: using /usr/bin/convert-im6.q16 to provide /usr/bin/convert (convert) in auto mode
update-alternatives: using /usr/bin/convert-im6.q16 to provide /usr/bin/convert-im6 (convert-im6) in auto mode
update-alternatives: using /usr/bin/composite-im6.q16 to provide /usr/bin/composite (composite) in auto mode
update-alternatives: using /usr/bin/composite-im6.q16 to provide /usr/bin/composite-im6 (composite-im6) in auto mode
update-alternatives: using /usr/bin/conjure-im6.q16 to provide /usr/bin/conjure (conjure) in auto mode
update-alternatives: using /usr/bin/conjure-im6.q16 to provide /usr/bin/conjure-im6 (conjure-im6) in auto mode
update-alternatives: using /usr/bin/import-im6.q16 to provide /usr/bin/import (import) in auto mode
update-alternatives: using /usr/bin/import-im6.q16 to provide /usr/bin/import-im6 (import-im6) in auto mode
update-alternatives: using /usr/bin/identify-im6.q16 to provide /usr/bin/identify (identify) in auto mode
update-alternatives: using /usr/bin/identify-im6.q16 to provide /usr/bin/identify-im6 (identify-im6) in auto mode
update-alternatives: using /usr/bin/stream-im6.q16 to provide /usr/bin/stream (stream) in auto mode
update-alternatives: using /usr/bin/stream-im6.q16 to provide /usr/bin/stream-im6 (stream-im6) in auto mode
update-alternatives: using /usr/bin/display-im6.q16 to provide /usr/bin/display (display) in auto mode
update-alternatives: using /usr/bin/display-im6.q16 to provide /usr/bin/display-im6 (display-im6) in auto mode
update-alternatives: using /usr/bin/montage-im6.q16 to provide /usr/bin/montage (montage) in auto mode
update-alternatives: using /usr/bin/montage-im6.q16 to provide /usr/bin/montage-im6 (montage-im6) in auto mode
update-alternatives: using /usr/bin/mogrify-im6.q16 to provide /usr/bin/mogrify (mogrify) in auto mode
update-alternatives: using /usr/bin/mogrify-im6.q16 to provide /usr/bin/mogrify-im6 (mogrify-im6) in auto mode
Setting up fonts-droid-fallback (1:6.0.1r16-1.1build1) ...
Setting up libdjvulibre-text (3.5.28-2ubuntu0.24.04.1) ...
Setting up libjxr-tools (1.2~git20170615.f752187-5.1ubuntu2) ...
Setting up libdjvulibre21:amd64 (3.5.28-2ubuntu0.24.04.1) ...
Setting up xfonts-utils (1:7.7+6build3) ...
Setting up libopenexr-3-1-30:amd64 (3.1.5-5.1build3) ...
Setting up imagemagick (8:6.9.12.98+dfsg1-5.2build2) ...
Setting up libmagickcore-6.q16-7-extra:amd64 (8:6.9.12.98+dfsg1-5.2build2) ...
Setting up fonts-urw-base35 (20200910-8) ...
Setting up libgs10-common (10.02.1~dfsg1-0ubuntu7.8) ...
Setting up libgs10:amd64 (10.02.1~dfsg1-0ubuntu7.8) ...
Setting up ghostscript (10.02.1~dfsg1-0ubuntu7.8) ...
Processing triggers for fontconfig (2.15.0-1.1ubuntu2) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for libc-bin (2.39-0ubuntu8.6) ...
Processing triggers for man-db (2.12.0-4build2) ...
Not building database; man-db/auto-update is not 'true'.

Running kernel seems to be up-to-date.

No services need to be restarted.

No containers need to be restarted.

No user sessions are running outdated binaries.

No VM guests are running outdated hypervisor (qemu) binaries on this host.
Collecting nbconvert
  Downloading nbconvert-7.16.6-py3-none-any.whl.metadata (8.5 kB)
Collecting beautifulsoup4 (from nbconvert)
  Downloading beautifulsoup4-4.14.2-py3-none-any.whl.metadata (3.8 kB)
Collecting bleach!=5.0.0 (from bleach[css]!=5.0.0->nbconvert)
  Downloading bleach-6.3.0-py3-none-any.whl.metadata (31 kB)
Collecting defusedxml (from nbconvert)
  Downloading defusedxml-0.7.1-py2.py3-none-any.whl.metadata (32 kB)
Collecting jinja2>=3.0 (from nbconvert)
  Downloading jinja2-3.1.6-py3-none-any.whl.metadata (2.9 kB)
Collecting jupyter-core>=4.7 (from nbconvert)
  Downloading jupyter_core-5.9.1-py3-none-any.whl.metadata (1.5 kB)
Collecting jupyterlab-pygments (from nbconvert)
  Downloading jupyterlab_pygments-0.3.0-py3-none-any.whl.metadata (4.4 kB)
Collecting markupsafe>=2.0 (from nbconvert)
  Downloading markupsafe-3.0.3-cp313-cp313-manylinux2014_x86_64.manylinux_2_17_x86_64.manylinux_2_28_x86_64.whl.metadata (2.7 kB)
Collecting mistune<4,>=2.0.3 (from nbconvert)
  Downloading mistune-3.1.4-py3-none-any.whl.metadata (1.8 kB)
Collecting nbclient>=0.5.0 (from nbconvert)
  Downloading nbclient-0.10.2-py3-none-any.whl.metadata (8.3 kB)
Collecting nbformat>=5.7 (from nbconvert)
  Downloading nbformat-5.10.4-py3-none-any.whl.metadata (3.6 kB)
Collecting packaging (from nbconvert)
  Downloading packaging-25.0-py3-none-any.whl.metadata (3.3 kB)
Collecting pandocfilters>=1.4.1 (from nbconvert)
  Downloading pandocfilters-1.5.1-py2.py3-none-any.whl.metadata (9.0 kB)
Collecting pygments>=2.4.1 (from nbconvert)
  Downloading pygments-2.19.2-py3-none-any.whl.metadata (2.5 kB)
Collecting traitlets>=5.1 (from nbconvert)
  Downloading traitlets-5.14.3-py3-none-any.whl.metadata (10 kB)
Collecting webencodings (from bleach!=5.0.0->bleach[css]!=5.0.0->nbconvert)
  Downloading webencodings-0.5.1-py2.py3-none-any.whl.metadata (2.1 kB)
Collecting tinycss2<1.5,>=1.1.0 (from bleach[css]!=5.0.0->nbconvert)
  Downloading tinycss2-1.4.0-py3-none-any.whl.metadata (3.0 kB)
Collecting platformdirs>=2.5 (from jupyter-core>=4.7->nbconvert)
  Downloading platformdirs-4.5.0-py3-none-any.whl.metadata (12 kB)
Collecting jupyter-client>=6.1.12 (from nbclient>=0.5.0->nbconvert)
  Downloading jupyter_client-8.6.3-py3-none-any.whl.metadata (8.3 kB)
Collecting python-dateutil>=2.8.2 (from jupyter-client>=6.1.12->nbclient>=0.5.0->nbconvert)
  Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
Collecting pyzmq>=23.0 (from jupyter-client>=6.1.12->nbclient>=0.5.0->nbconvert)
  Downloading pyzmq-27.1.0-cp312-abi3-manylinux_2_26_x86_64.manylinux_2_28_x86_64.whl.metadata (6.0 kB)
Collecting tornado>=6.2 (from jupyter-client>=6.1.12->nbclient>=0.5.0->nbconvert)
  Downloading tornado-6.5.2-cp39-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.8 kB)
Collecting fastjsonschema>=2.15 (from nbformat>=5.7->nbconvert)
  Downloading fastjsonschema-2.21.2-py3-none-any.whl.metadata (2.3 kB)
Collecting jsonschema>=2.6 (from nbformat>=5.7->nbconvert)
  Downloading jsonschema-4.25.1-py3-none-any.whl.metadata (7.6 kB)
Collecting attrs>=22.2.0 (from jsonschema>=2.6->nbformat>=5.7->nbconvert)
  Downloading attrs-25.4.0-py3-none-any.whl.metadata (10 kB)
Collecting jsonschema-specifications>=2023.03.6 (from jsonschema>=2.6->nbformat>=5.7->nbconvert)
  Downloading jsonschema_specifications-2025.9.1-py3-none-any.whl.metadata (2.9 kB)
Collecting referencing>=0.28.4 (from jsonschema>=2.6->nbformat>=5.7->nbconvert)
  Downloading referencing-0.37.0-py3-none-any.whl.metadata (2.8 kB)
Collecting rpds-py>=0.7.1 (from jsonschema>=2.6->nbformat>=5.7->nbconvert)
  Downloading rpds_py-0.29.0-cp313-cp313-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (4.1 kB)
Collecting six>=1.5 (from python-dateutil>=2.8.2->jupyter-client>=6.1.12->nbclient>=0.5.0->nbconvert)
  Downloading six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
Collecting soupsieve>1.2 (from beautifulsoup4->nbconvert)
  Downloading soupsieve-2.8-py3-none-any.whl.metadata (4.6 kB)
Collecting typing-extensions>=4.0.0 (from beautifulsoup4->nbconvert)
  Downloading typing_extensions-4.15.0-py3-none-any.whl.metadata (3.3 kB)
Downloading nbconvert-7.16.6-py3-none-any.whl (258 kB)
Downloading mistune-3.1.4-py3-none-any.whl (53 kB)
Downloading bleach-6.3.0-py3-none-any.whl (164 kB)
Downloading tinycss2-1.4.0-py3-none-any.whl (26 kB)
Downloading jinja2-3.1.6-py3-none-any.whl (134 kB)
Downloading jupyter_core-5.9.1-py3-none-any.whl (29 kB)
Downloading markupsafe-3.0.3-cp313-cp313-manylinux2014_x86_64.manylinux_2_17_x86_64.manylinux_2_28_x86_64.whl (22 kB)
Downloading nbclient-0.10.2-py3-none-any.whl (25 kB)
Downloading jupyter_client-8.6.3-py3-none-any.whl (106 kB)
Downloading nbformat-5.10.4-py3-none-any.whl (78 kB)
Downloading fastjsonschema-2.21.2-py3-none-any.whl (24 kB)
Downloading jsonschema-4.25.1-py3-none-any.whl (90 kB)
Downloading attrs-25.4.0-py3-none-any.whl (67 kB)
Downloading jsonschema_specifications-2025.9.1-py3-none-any.whl (18 kB)
Downloading pandocfilters-1.5.1-py2.py3-none-any.whl (8.7 kB)
Downloading platformdirs-4.5.0-py3-none-any.whl (18 kB)
Downloading pygments-2.19.2-py3-none-any.whl (1.2 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 149.5 MB/s  0:00:00
Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
Downloading pyzmq-27.1.0-cp312-abi3-manylinux_2_26_x86_64.manylinux_2_28_x86_64.whl (840 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 841.0/841.0 kB 98.3 MB/s  0:00:00
Downloading referencing-0.37.0-py3-none-any.whl (26 kB)
Downloading rpds_py-0.29.0-cp313-cp313-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (395 kB)
Downloading six-1.17.0-py2.py3-none-any.whl (11 kB)
Downloading tornado-6.5.2-cp39-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (443 kB)
Downloading traitlets-5.14.3-py3-none-any.whl (85 kB)
Downloading webencodings-0.5.1-py2.py3-none-any.whl (11 kB)
Downloading beautifulsoup4-4.14.2-py3-none-any.whl (106 kB)
Downloading soupsieve-2.8-py3-none-any.whl (36 kB)
Downloading typing_extensions-4.15.0-py3-none-any.whl (44 kB)
Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
Downloading jupyterlab_pygments-0.3.0-py3-none-any.whl (15 kB)
Downloading packaging-25.0-py3-none-any.whl (66 kB)
Installing collected packages: webencodings, fastjsonschema, typing-extensions, traitlets, tornado, tinycss2, soupsieve, six, rpds-py, pyzmq, pygments, platformdirs, pandocfilters, packaging, mistune, markupsafe, jupyterlab-pygments, defusedxml, bleach, attrs, referencing, python-dateutil, jupyter-core, jinja2, beautifulsoup4, jupyter-client, jsonschema-specifications, jsonschema, nbformat, nbclient, nbconvert

Successfully installed attrs-25.4.0 beautifulsoup4-4.14.2 bleach-6.3.0 defusedxml-0.7.1 fastjsonschema-2.21.2 jinja2-3.1.6 jsonschema-4.25.1 jsonschema-specifications-2025.9.1 jupyter-client-8.6.3 jupyter-core-5.9.1 jupyterlab-pygments-0.3.0 markupsafe-3.0.3 mistune-3.1.4 nbclient-0.10.2 nbconvert-7.16.6 nbformat-5.10.4 packaging-25.0 pandocfilters-1.5.1 platformdirs-4.5.0 pygments-2.19.2 python-dateutil-2.9.0.post0 pyzmq-27.1.0 referencing-0.37.0 rpds-py-0.29.0 six-1.17.0 soupsieve-2.8 tinycss2-1.4.0 tornado-6.5.2 traitlets-5.14.3 typing-extensions-4.15.0 webencodings-0.5.1
Configuration file: /home/runner/work/fandaren.github.io/fandaren.github.io/_config.yml
            Source: /home/runner/work/fandaren.github.io/fandaren.github.io
       Destination: /home/runner/work/fandaren.github.io/fandaren.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
DEPRECATION WARNING: `to_time` will always preserve the receiver timezone rather than system local time in Rails 8.1.To opt in to the new behavior, set `config.active_support.to_time_preserves_timezone = :zone`. (called from load at /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/bin/jekyll:25)
Fetching external posts from medium.com:
                    ------------------------------------------------
      Jekyll 4.4.1   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/feedjira-3.2.5/lib/feedjira.rb:59:in `parse': No valid parser for XML. (Feedjira::NoParserAvailable)
	from /home/runner/work/fandaren.github.io/fandaren.github.io/_plugins/external-posts.rb:28:in `fetch_from_rss'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/_plugins/external-posts.rb:17:in `block in generate'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/_plugins/external-posts.rb:14:in `each'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/_plugins/external-posts.rb:14:in `generate'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:193:in `block in generate'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:191:in `each'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:191:in `generate'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/site.rb:79:in `process'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:28:in `process_site'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:65:in `build'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:36:in `process'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in `each'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-4.4.1/exe/jekyll:15:in `<top (required)>'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/bin/jekyll:25:in `load'
	from /home/runner/work/fandaren.github.io/fandaren.github.io/vendor/bundle/ruby/3.3.0/bin/jekyll:25:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/cli/exec.rb:59:in `load'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/cli/exec.rb:59:in `kernel_load'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/cli/exec.rb:23:in `run'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/cli.rb:452:in `exec'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/vendor/thor/lib/thor.rb:538:in `dispatch'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/cli.rb:35:in `dispatch'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/cli.rb:29:in `start'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/exe/bundle:28:in `block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/lib/ruby/gems/3.3.0/gems/bundler-2.6.9/exe/bundle:20:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/bin/bundle:25:in `load'
	from /opt/hostedtoolcache/Ruby/3.3.5/x64/bin/bundle:25:in `<main>'
Error: Process completed with exit code 1.
```


### What operating system are you using?

Not applicable (e.g. you're using GitHub Pages or other hosting)

### Where are you seeing the problem on?

Deployed site

### More info

<img width="3016" height="1062" alt="Image" src="https://github.com/user-attachments/assets/2d442e16-4b9a-436f-ad31-58e275ccea19" />

--- a/_plugins/external-posts.rb
+++ b/_plugins/external-posts.rb
@@ -25,7 +25,12 @@ module ExternalPosts
     def fetch_from_rss(site, src)
       xml = HTTParty.get(src['rss_url']).body
       return if xml.nil?
-      feed = Feedjira.parse(xml)
+      begin
+        feed = Feedjira.parse(xml)
+      rescue StandardError => e
+        puts "Error parsing RSS feed from #{src['rss_url']} - #{e.message}"
+        return
+      end
       process_entries(site, src, feed.entries)
     end
 

